pkg/cmd/server/start/start_master.go

```diff
@@ -551,8 +551,10 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunNodeController()
 		kc.RunScheduler()
 		kc.RunReplicationController(rcClient)
-		kc.RunJobController(jobClient)
-		kc.RunHPAController(hpaOClient, hpaKClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
+		if kc.ControllerManager != nil && kc.ControllerManager.EnableExperimental {
```

Comment:
@deads2k do you think we should belt-and-suspenders by checking `kc.Master.EnableExp && ...`?